src/data/models/Battle.js

```diff
@@ -33,7 +33,9 @@ const Battle = Model.define(
       allowNull: true,
       defaultValue: '0000-00-00 00:00:00',
       get() {
-        return moment(this.getDataValue('Started')).format('X');
+        return moment(this.getDataValue('Started'))
+          .add(8, 'hours')
```

Comment:
Can you move this timezone correction to other place, maybe to the view you are displaying the info? I didn't check but I believe this will break the chatlog feature on the battle page as it uses the start time to fetch the correct lines. If we start correcting the timezone issues like this, it feels like it might result in unnecessary juggling back and forth between the server and client. I think the timezone correction could be done purely on the view layer.

Reply:
The idea was to get a proper unix timestamp this way, since the datetime in db is saved in utc-8 without a timezone indicator. Of course yea it gives an issue when you use this value to query for data, I'm trying another solution.

Reply:
Yeah I see. Maybe we could start using moment.js timezones, have a look at this: https://momentjs.com/timezone/docs/#/using-timezones/. It saves the timezone information in the object and then it would be easy to display the time in whatever user-specified timezone setting in the front end.